example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -21,11 +21,12 @@
  * Does not post to CMR (that is in a separate test)
  */
 
-const { Execution, Pdr } = require('@cumulus/api/models');
```

Comment:
Nit: can we also remove this in the `beforeAll`:

```
process.env.PdrsTable = `${config.stackName}-PdrsTable`;
```

Just want to clear all traces of reference to the Dynamo tables/models. We would need to do the same in all of these tests

Reply:
I did do so for most of them. I must have missed it here.

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/7ed58e6a1895c91c2d21825e119ebfa9ace452bc